src/lib/falcor/routes/articles/__tests__/database-calls.test.js

```diff
@@ -55,3 +60,87 @@ describe('getPaginatedArticle', () => {
     await expect(getPaginatedArticle(database, 0, 5)).resolves.toEqual([]);
   });
 });
+
+describe('createNewArticle', () => {
+  const insertMock = jest.fn().mockResolvedValue([1]);
+  const databaseMock = jest.fn().mockReturnValue({ insert: insertMock });
+
+  beforeAll(() => {
+    MockDate.set(1524998880085);
```

Comment:
Just anything that's a constant. It was the time I wrote the test. It's just so `new Date()` returns something constant we can assert instead of the time the test is running. I can add a comment